zwave_js_server/model/driver.py

```diff
@@ -29,3 +32,18 @@ def receive_event(self, event: Event) -> None:
 
     def handle_all_nodes_ready(self, event: Event) -> None:
         """Process a driver all nodes ready event."""
+
+    async def async_update_log_config(self, log_config: LogConfig) -> bool:
+        """Update log config for driver."""
+        result = await self.client.async_send_command(
+            {
+                "command": "update_log_config",
+                "config": {
+                    snake_to_camel_case(field.name): getattr(log_config, field.name)
```

Comment:
This seems a little hacky. Shouldn't we just require the `LogConfig` class to use the correct formatting that ZJS Server expects? And otherwise maybe just manually rewrite `logToFile` and `forceConsole` instead of all this logic for 2 fields 😬

Reply:
yep we can do that, I over thought this one and was thinking that this could be used elsewhere, but better to make things more explicit so that we can catch if/when the model changes